docs/qcfractal/source/managers_faq.rst

```diff
@@ -0,0 +1,137 @@
+Queue Manager Frequent Questions and Issues
+===========================================
+
+This page documents some of the frequent questions and issues we see with the
+Queue Managers. If this page and none of the other documentation pages have
+answered your question, please ask `on GitHub <https://github.com/MolSSI/QCFractal/>`_ or
+`join our Slack group <https://join.slack.com/t/qcdb/shared_invite/enQtNDIzNTQ2OTExODk0LWM3OTgxN2ExYTlkMTlkZjA0OTExZDlmNGRlY2M4NWJlNDlkZGQyYWUxOTJmMzc3M2VlYzZjMjgxMDRkYzFmOTE>`_
```

Comment:
It would be good to make an alias for this at the sphinx level so we can just to `_slack_inivite_url` or similar. I forget how to do this off the top of my head however.

Reply:
`conf.py` https://github.com/psi4/psi4/blob/master/doc/sphinxman/source/conf.py.in#L342